README.md

```diff
@@ -10,6 +10,33 @@ Fake for Seam Devicedb. Use with Fake Seam Connect.
 This module allows you to spin up an in-memory server
 for the Seam Devicedb.
 
+## 📓 Notes
+
+### Text Search
+
+Full-text search on real devicedb API (using the `?text_search` parameter) is far more flexible than the simple matching provided by this fake.
+
+### Seeding with Data
+
+This fake can be seeded with data from the live API. To do so, either call `POST /_fake/populate-from-live-api` or use the exported helper:
```

Comment:
Instead of populate (introducing a new concept), can we just call this `seedFromLiveApi`